docs/v0.5/en/db-cloud-shared/clients/otlp-integration.md

```diff
@@ -7,7 +7,7 @@ To send OpenTelemetry Metrics to GreptimeDB through OpenTelemetry SDK libraries,
 * URL: `https://<host>/v1/otlp/v1/metrics`
 * Headers:
   * `X-Greptime-DB-Name`: `<dbname>`
-  * `Authorization`: `Basic` authentication, which is a Base64 encoded string of `<username>:<password>`. For more information, please refer to [Authentication](https://docs.greptime.com/user-guide/clients/authentication) and [HTTP API](https://docs.greptime.com/user-guide/clients/http-api#authentication)
```

Comment:
The links in dictionaries `db-cloud-shared` and `greptimecloud` need to be preserved because the content is used in the console of [greptime.cloud](https://greptime.cloud/), that means the paths cannot be resolved.

Reply:
OK. Let me revert this change. Although, these pages doesn't seems to be navigatable in the sidebar, perhaps we can hold them outside the docs?

Reply:
Besides, is zh version affected?

Reply:
Updated.